lib/tapioca/generator.rb

```diff
@@ -559,6 +560,38 @@ def compile_gem_rbi(gem)
       end
     end
 
+    sig { params(gem: Gemfile::Gem, generated_content: String).returns(String) }
+    def merge_with_copied_rbi(gem, generated_content)
+      return generated_content unless File.directory?("#{gem.full_gem_path}/rbi")
+
+      generated_rbi = RBI::Parser.parse_string(generated_content)
+      copied_rbi = copy_gem_rbi(gem)
+      display_conflict_warnings(generated_rbi, copied_rbi)
+      final_rbi = RBI::Rewriters::Merge.merge_trees(generated_rbi, copied_rbi, keep: RBI::Rewriters::Merge::Keep::LEFT)
+      final_rbi.string
+    end
+
+    sig { params(gem: Gemfile::Gem).returns(RBI::Tree) }
+    def copy_gem_rbi(gem)
+      rbi = RBI::Tree.new
+      Dir.glob("#{gem.full_gem_path}/rbi/*.rbi").each do |file|
+        gem_rbi = RBI::Parser.parse_file(file)
+        display_conflict_warnings(rbi, gem_rbi)
```

Comment:
Can we add some conflicts to the `bar` gem RBI files, so that we can expect to see these warnings in the test output, as well?